src/HelperFunctions.tsx

```diff
@@ -0,0 +1,23 @@
+import { FilterBySelect } from './types/FilterBySelect';
+import { Todo } from './types/Todo';
+
+export function filterTodos(todos: Todo[], filterBy: FilterBySelect): Todo[] {
```

Comment:
Consider using rest parameters instead of passing an array as an argument for better readability 📘.